js/index.js

```diff
@@ -135,9 +134,39 @@ form.addEventListener('submit', (event) => {
   if (!emailRegExp.test(email.value.trim())) {
     form.submit();
   } else {
-
     event.preventDefault();
     //emailClass.classList.toggle('error');
-    errMessageZone.innerText = 'Please enter an entirely lower case email, please';
+    errMessageZone.innerText =
+      'Please enter an entirely lower case email, please';
   }
 });
+
+// preserving Data
+
+let data = {};
+
+form.addEventListener('input', function (e) {
+  switch (e.target.id) {
+    case 'fullName':
+      data.fullName = e.target.value;
+      console.log('hhhhhhhhhhhhhhhhhhhhhhhh');
```

Comment:
- [ ] Please do not leave `console.log()` in your codebase because it is considered as a bad practice. 👌